packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -3047,7 +3090,22 @@ RED.view = (function() {
                                             l = d.type;
                                         }
                                     }
-                                    return l;
+                                    var sa = convertLineBreakCharacter(l);
```

Comment:
This is a lot of extra work that is now done whenever a node is redrawn. We may need to look at caching the result and only rebuilding the content when we see it has changed.